### PR TITLE
[DT-801] docs: Updating README and env sample files

### PR DESCRIPTION
**Pull request by crazeekennee**
**Background**

The resolution-service's README.md file has gotten out of date, and needs some maintenance to be current with our resolution-service

**Changes**

We updated the README.md file to reflect the currently required environment variables, recommended service versions, service descriptions, and removed some ambiguity in the service description.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,25 +56,27 @@ variety of platforms and in the cloud.
 
 ```
 NODE_ENV=production
-RESOLUTION_POSTGRES_HOST=example.com:5432      # DB host
-RESOLUTION_POSTGRES_USERNAME=example           # DB user configured in postgres
-RESOLUTION_POSTGRES_PASSWORD=password          # DB password configured in postgres
+RESOLUTION_POSTGRES_HOST=example.com          	# DB host
+RESOLUTION_POSTGRES_PORT=5432                   # DB port
```
Comment (crazeekennee) [inline README.md:60]: Specifying RESOLUTION_POSTGRES_PORT separately from the host line,, to match convention with line 99.

File: README.md [hunk 2]
```diff
@@ -209,6 +212,7 @@ Additional pre-requirements that are necessary for development:
 1. Install project dependencies
 
 ```
+nvm install 14.16.1
```
Comment (crazeekennee) [inline README.md:215]: This step was required for me, but I only knew about it after running `nvm use`, so why not specify it before, so it'll be installed before `nvm use` tries to use it ?

File: README.md [hunk 3]
```diff
@@ -225,6 +229,10 @@ RESOLUTION_POSTGRES_DATABASE=resolution_service
 ETHEREUM_JSON_RPC_API_URL=localhost:8545
 POLYGON_JSON_RPC_API_URL=localhost:8546
 VIEWBLOCK_API_KEY=apikey
+METADATA_API=apikey
```
Comment (crazeekennee) [inline README.md:232]: Same comment as on line 67, application wouldn't start unless these value existed in local.dev.env

**Review by sammyluo — APPROVED** [review]
Looks great! Thanks for updating it! 🚀 ⭐

**Pull request merged**